contributors/devel/sig-instrumentation/migration-to-structured-logging.md

```diff
@@ -95,50 +103,21 @@ Structured logging functions follow a different logging interface design than ot
 minimal design from [logr] thus there is no one-to-one mapping.
 
 Simplified mapping between functions:
-* `klog.Infof`, `klog.Info`, `klog.Infoln`, `klog.InfoDepth` -> `klog.InfoS`
+* `klog.Infof`, `klog.Info`, `klog.Infoln` -> `klog.InfoS`
+* `klog.InfoDepth` -> `klog.InfoSDepth`
 * `klog.V(N).Infof`, `klog.V(N).Info`, `klog.V(N).Infoln` -> `klog.V(N).InfoS`
-* `klog.Warning`, `klog.Warningf`, `klog.Warningln`, `klog.WarningDepth` -> `klog.InfoS`
+* `klog.Warning`, `klog.Warningf`, `klog.Warningln` -> `klog.InfoS`
+* `klog.WarningDepth` -> `klog.InfoSDepth`
 * `klog.V(N).Warning`, `klog.V(N).Warningf`, `klog.V(N).Warningln`, `klog.V(N).WarningDepth` -> `klog.V(N).InfoS`
```

Comment:
Let's use this occasion to clean up the incorrect examples from the original guide to make it more readable. There is no `klog.V(N).*` other then for `Info*`. There are not `klog.V(N).Warning*`, no `klog.V(N).Error*` and no `klog.V(N).Fatal`. Can you remove them ?

Reply:
OK, Let me do it